dialer.go

```diff
@@ -625,14 +631,18 @@ type instrumentedConn struct {
 	instance       string
 	metricRecorder telv2.MetricRecorder
 	attrs          telv2.Attributes
+	bytesRead      atomic.Int64
+	bytesWritten   atomic.Int64
+	reportTicker   *time.Ticker
+	stopReporter   func()
 }
 
 // Read delegates to the underlying net.Conn interface and records number of
 // bytes read.
 func (i *instrumentedConn) Read(b []byte) (int, error) {
 	bytesRead, err := i.Conn.Read(b)
 	if err == nil {
-		go tel.RecordBytesReceived(context.Background(), int64(bytesRead), i.instance, i.dialerID)
+		i.bytesRead.Add(int64(bytesRead))
 		go i.metricRecorder.RecordBytesRxCount(context.Background(), int64(bytesRead), i.attrs)
```

Comment:
This will need to move into the report goroutine too. Same with RecordBytesTxCount too.

Reply:
Done